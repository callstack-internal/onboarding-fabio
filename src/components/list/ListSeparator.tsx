import React from 'react';
import {StyleSheet, View} from 'react-native';
import Colors from '../../styles/Colors';

const ListSeparator = () => {
  return <View style={styles.rootContainer} />;
};

const styles = StyleSheet.create({
  rootContainer: {
    backgroundColor: Colors.lightGray,
    height: 1,
  },
});

export default ListSeparator;
