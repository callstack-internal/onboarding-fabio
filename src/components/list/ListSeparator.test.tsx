import React from 'react';
import renderer from 'react-test-renderer';
import ListSeparator from './ListSeparator';

test('should render correctly', () => {
  const tree = renderer.create(<ListSeparator />).toJSON();

  expect(tree).toMatchSnapshot();
});
