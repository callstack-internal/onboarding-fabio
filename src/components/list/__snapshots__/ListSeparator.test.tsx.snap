// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#D3D3D3",
      "height": 1,
    }
  }
/>
`;
